src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/StorageSection/CustomizeSource/utils.ts

```diff
@@ -22,7 +25,11 @@ import {
 } from '../constants';
 
 export const getQuantityFromSource = (source: V1beta1DataVolumeSpec) =>
-  source?.storage?.resources?.requests?.storage;
+  hasSizeUnit(source?.storage?.resources?.requests?.storage)
+    ? removeByteSuffix(source?.storage?.resources?.requests?.storage)
+    : removeByteSuffix(
+        xbytes(Number(source?.storage?.resources?.requests?.storage), { iec: true, space: false }),
+      );
```

Comment:
```suggestion
  { 
  const storage = source?.storage?.resources?.requests?.storage;
  const value = hasSizeUnit(storage) ? storage : xbytes(Number(storage), { iec: true, space: false })
  
    return  removeByteSuffix(value);
  }
```

What do you think? IMO its more clear

Reply:
Yees! I wanted to do this. Just forgot before pushing the code to GH :D